spec/requests/api/v2/providers_spec.rb

```diff
@@ -236,6 +240,46 @@
       end
     end
 
+    context "with the maximum number of sites" do
+      let!(:provider) { create(:provider, course_count: 0, site_count: Site::POSSIBLE_CODES.size, organisations: [organisation]) }
+
+      before do
+        get "/api/v2/providers/#{provider.provider_code}",
+          headers: { 'HTTP_AUTHORIZATION' => credentials }
+      end
+
+      it 'has a data section with the correct attributes' do
+        json_response = JSON.parse(response.body)
+        expect(json_response).to eq(
```

Comment:
I wonder whether we should only assert the `can_add_more_sites?` on this response, to make the spec more explicit and less prone to breakage if unrelated things change?

Reply:
Agree